src/data/nat/lattice.lean

```diff
@@ -105,6 +108,36 @@ noncomputable instance : conditionally_complete_linear_order_bot ℕ :=
   .. (infer_instance : order_bot ℕ), .. (lattice_of_linear_order : lattice ℕ),
   .. (infer_instance : linear_order ℕ) }
 
+lemma Inf_add {n : ℕ} {p : ℕ → Prop} (hn : n ≤ Inf (set_of p)) :
+  Inf {m : ℕ | p (m + n)} + n = Inf (set_of p) :=
```

Comment:
```suggestion
lemma Inf_add {n : ℕ} {p : ℕ → Prop} (hn : n ≤ Inf {m | p m}) :
  Inf {m : ℕ | p (m + n)} + n = Inf {m | p m} :=
```
We don't often use the function `set_of` directly, but rather its notation.
Other than that, this LGTM.
bors d+